weaviate/collections/classes/cluster.py

```diff
@@ -73,7 +73,10 @@ def nodes_verbose(nodes: List[NodeREST]) -> List[Node[Shards, Stats]]:
                         shard_count=node["stats"]["shardCount"],
                     )
                     if "stats" in node
-                    else []
+                    else Stats(
+                        object_count=0,
+                        shard_count=0,
+                    )
```

Comment:
why this change?

Reply:
`pyright` didn't like `[]` being returned where `Stats` is expected. The previous version before the change on `main` must have missed it